### PR TITLE
The Bengal Tiger

### PR DESCRIPTION
**Pull request by ch1771dr0b0**

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, can you find a horizontal image close to 600x400px in size? Also, please use hyphens, instead of the underscores. Thanks.

**Comment by ch1771dr0b0** [comment]
> Hi, can you find a horizontal image close to 600x400px in size? Also, please use hyphens, instead of the underscores. Thanks.

completed the changes

File: src/recipes/the-bengal-tiger.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    ],
+    "directions": [
+        "In a shaker filled with ice pour the spirits, some sweet and sour mix",
+        "Pour the juices and shake the mixture.",
+        "In a tall glass filled with ice pour the drink.",
+        "Garnish with a wedge of gondoraj lemon and twist a peel of the lemon over the drink.",
+        "Rub the peel on the glass for the freshness and aroma.",
+        "Serve with a smile!"
+    ],
+    "image": "the_bengal_tiger.jpg",
```
Comment (alfg) [inline src/recipes/the-bengal-tiger.json:40]: Need to update here as well.

**Review by ch1771dr0b0 — COMMENTED** [review]
completed the changes to json file.

File: src/recipes/the-bengal-tiger.json (hunk 1 above)
Comment (ch1771dr0b0) [reply src/recipes/the-bengal-tiger.json]: change image name in json file.

**Comment by alfg** [comment]
> 
> 
> > Hi, can you find a horizontal image close to 600x400px in size? Also, please use hyphens, instead of the underscores. Thanks.
> 
> completed the changes

Hi, the image doesn't seem to be horizontal.

**Comment by ch1771dr0b0** [comment]
> > > Hi, can you find a horizontal image close to 600x400px in size? Also, please use hyphens, instead of the underscores. Thanks.
> > 
> > 
> > completed the changes
> 
> Hi, the image doesn't seem to be horizontal.

changes completed.

**Comment by alfg** [comment]
> 
> 
> > > > Hi, can you find a horizontal image close to 600x400px in size? Also, please use hyphens, instead of the underscores. Thanks.
> > > 
> > > 
> > > completed the changes
> > 
> > 
> > Hi, the image doesn't seem to be horizontal.
> 
> changes completed.

Please provide a horizontal image. This one is still vertical.

**Comment by ch1771dr0b0** [comment]
> > > > > Hi, can you find a horizontal image close to 600x400px in size? Also, please use hyphens, instead of the underscores. Thanks.
> > > > 
> > > > 
> > > > completed the changes
> > > 
> > > 
> > > Hi, the image doesn't seem to be horizontal.
> > 
> > 
> > changes completed.
> 
> Please provide a horizontal image. This one is still vertical.

Uploa

> > > > > Hi, can you find a horizontal image close to 600x400px in size? Also, please use hyphens, instead of the underscores. Thanks.
> > > > 
> > > > 
> > > > completed the changes
> > > 
> > > 
> > > Hi, the image doesn't seem to be horizontal.
> > 
> > 
> > changes completed.
> 
> Please provide a horizontal image. This one is still vertical.

Uploaded new image. Please verify.

**Comment by alfg** [comment]
This is good. Thanks!

**Pull request merged**